todolist/settings.py

```diff
@@ -66,7 +66,7 @@
         'NAME': 'app_db',
         'USER': 'app_user',
         'PASSWORD': '1234',
```

Comment:
The database password is hardcoded, which is a security risk. Consider using environment variables or a configuration management tool to handle sensitive information.